native/include/mixpanel/mixpanel.hpp

```diff
@@ -228,6 +229,9 @@ namespace mixpanel
         private:
             friend class People;
             friend class mixpanel::detail::Worker;
+            FRIEND_TEST(MixpanelNetwork, RetryAfter);
+            FRIEND_TEST(MixpanelNetwork, BackOffTime);
+            FRIEND_TEST(MixpanelNetwork, FailureRecovery);
```

Comment:
Not 100% sure, but I suspect that if you made these lines e.g.

    FRIEND_TEST(::MixpanelNetwork, RetryAfter);

then you could remove the `namespace mixpanel` and accompanying note in "test_network.cpp".